tools/run_tests/run_interop_tests.py

```diff
@@ -1285,8 +1297,10 @@ def aggregate_http2_results(stdout):
 
     jobs = []
     if args.cloud_to_prod:
-        if args.transport_security != 'tls':
-            print('TLS is always enabled for cloud_to_prod scenarios.')
+        if args.transport_security != 'tls' and args.transport_security != 'google_default_credentials':
```

Comment:
nit:  this could be slightly cleaned up to `if args.transport_security not in ['tls', 'google_default_credentials']:`

Reply:
Good point.